test/ingress/aws/aws_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws
+// +build e2e
+
+package aws
```

Comment:
Sorry for the silly question, but I fail to see where this is used. Maybe in the ci project will be used once it is merged? No idea :D

Reply:
What do you mean by "this"?

Reply:
Exaclty that. Great, then, this is even added to the ci! Awesome! :) (sorry, not familiar with the CI still :see_no_evil: )